builder/deps.go

```diff
@@ -45,6 +45,7 @@ func (b *Deps) Write(writer io.Writer) (err error) {
 					Name:     d.Name,
 					Version:  d.Version,
 					SHA:      d.ResolvedIdentifier,
+					Labels:   d.Labels,
```

Comment:
Also, seems  the "provider" field on the dependency is always blank for any analysis.

Reply:
Fixed.